core/model/collection.py

```diff
@@ -74,13 +74,15 @@ class Collection(Base, HasSessionCache):
     external_integration_id = Column(
         Integer, ForeignKey("externalintegrations.id"), unique=True, index=True
     )
+    _external_integration: ExternalIntegration
 
     # A Collection may specialize some other Collection. For instance,
     # an Overdrive Advantage collection is a specialization of an
     # ordinary Overdrive collection. It uses the same access key and
     # secret as the Overdrive collection, but it has a distinct
     # external_account_id.
     parent_id = Column(Integer, ForeignKey("collections.id"), index=True)
+    parent: "Collection"
```

Comment:
Took me a while to figure out this is a backref placeholder, maybe add a comment to that effect